scripts/cover.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 
-docker-compose run --rm test cover -test -make 'prove -r t/; exit $?'
+# This script doesn't do the wait-for song and dance so if there are sporadic database errors,
```

Comment:
One option for getting this to be reliable w/o using `wait-for` is to use the `depends-on`, `healthcheck`, and `        condition: service_healthy` options in the docker-compose, e.g.

https://github.com/hathitrust/catalog/blob/main/docker-compose.yml#L56-L58
plus https://github.com/hathitrust/catalog/blob/main/docker-compose.yml#L20-L25

Reply:
I'll let this go for now since this script is just a convenience for development.